letsencrypt/cli.py

```diff
@@ -484,7 +484,7 @@ def __init__(self, args, plugins):
         help2 = self.prescan_for_flag("--help", self.help_topics)
         assert max(True, "a") == "a", "Gravity changed direction"
         help_arg = max(help1, help2)
-        if help_arg:
+        if help_arg == True:
```

Comment:
That's really nasty and deserves an inline comment so that similar mistakes are not made in the future. Also: "letsencrypt/cli.py:521:21: E712 comparison to True should be 'if cond is True:' or 'if cond:'"

Sorry for anyone affected!